### ISSUE TITLE
GUI - create pipeline need to change the concurrent Pipelines to object

### ISSUE THREAD

**Issue by tamir321** [kube-HPC/hkube#742]
**HKube micro-service**
To which micro-service the bug related to, `Dashboard` ,`Monitor-Server` etc.

**Describe the bug**
the concurrentPipeLines was change from number to object need to change the UI to support it
```json
 "options": {
    "ttl": 3600,
    "batchTolerance": 80,
    "concurrentPipelines": {
      "amount": 1,
      "rejectOnFailure": true
    },
    "progressVerbosityLevel": "info"
  },
```
![10.32.10.11_hkube_dashboard_ (1).png](https://images.zenhubusercontent.com/5ddb91c470bd590001fbdfae/2b915ee4-d9eb-4733-9e02-fd930ccc2e5c)
**Expected behavior**

**To Reproduce**

**Screenshots**

**Additional context**

**Comment by denvash** [comment]
Fixed on `v1.2.41`

**Comment by tamir321** [comment]
tested on 1.2.41
